modules/nw-metallb-advertise-address-pool-with-bgp-advanced.adoc

```diff
@@ -81,11 +81,11 @@ metadata:
   namespace: metallb-system
 spec:
   ipAddressPools:
-   - doc-example-bgp-adv
-  - communities:
+    - doc-example-bgp-adv
+  communities:
```

Comment:
Maybe add here a different local pref such as 200

Reply:
Why would adding such be relevant? It is just an example, it does not need to include all parameters.
The PR is about fixing syntax errors.

Reply:
The "communities" change is a syntax error that I am fixing in this PR. Its intended value is left unchanged.